UnitTesting/PlacedBets/PlacedBetsUnitTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿using AutoMapper;
+using Domain;
+using Domain.Dto.PlacedBet;
+using Domain.Entities;
+using Domain.ErrorEntities;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using Services;
+using Services.Interfaces;
+using Xunit.Abstractions;
+
+namespace UnitTesting.PlacedBets
```

Comment:
Are all tests consistently passing for you?
I tried to run the tests and always have some fail:

![image](https://github.com/user-attachments/assets/c9ea2cad-a71f-47e7-8cee-7f67816348e5)